backslash/contrib/slash_plugin.py

```diff
@@ -297,25 +297,14 @@ def is_session_exist(self, session_id):
 
     @handle_exceptions
     @slash.plugins.registers_on(None)
-    def get_tests_to_resume(self, session_id, get_successful=False, planned_first=False, failed_first=False):
+    def get_tests_to_resume(self, session_id, **kwargs):
```

Comment:
Add a docstring explaining the meaning of kwargs. It's also best to pass them not with `**`, as they will shadow any future parameter given to this function...